src/sync/sync_manager.hpp

```diff
@@ -104,19 +104,18 @@ friend class SyncSession;
 
     // Get a sync user for a given identity, or create one if none exists yet, and set its token.
     // If a logged-out user exists, it will marked as logged back in.
-    std::shared_ptr<SyncUser> get_user(const std::string& identity,
-                                       std::string refresh_token,
-                                       util::Optional<std::string> auth_server_url=none,
-                                       SyncUser::TokenType token_type=SyncUser::TokenType::Normal);
-    // Get an existing user for a given identity, if one exists and is logged in.
-    std::shared_ptr<SyncUser> get_existing_logged_in_user(const std::string& identity) const;
+    std::shared_ptr<SyncUser> get_user(const SyncUserIdentifier& identifier, std::string refresh_token);
```

Comment:
There's no replacement, but if bindings are using it it would be trivial to add it back in (and the tests), and make the few adjustments necessary to work.

@cmelchior, @nirinchev, @nhachicha are any of you using `get_existing_logged_in_user`? Sorry, I should have asked first.